src/differential_expression/deseq2/config.vsh.yaml

```diff
@@ -0,0 +1,150 @@
+  Note that this component only considers factors as explanatory variables, and excludes covariates from the analysis.
+
+authors:
+  - __merge__: /src/authors/jakub_majercik.yaml
+    roles: [ author ]
+  - __merge__: /src/authors/dorien_roosen.yaml
+    roles: [ author ]
+  - __merge__: /src/authors/dries_de_maeyer.yaml
+    roles: [ contributor ]
+  - __merge__: /src/authors/weiwei_schultz.yaml
+    roles: [ contributor ]
+
+argument_groups:
+  - name: Inputs
+    arguments:
+      - name: "--input"
+        alternatives: ["-i"]
+        type: file
+        description: Input h5mu file containing (pseudo-)bulk transcriptomic samples.
+        direction: input
+        required: true
+      - name: "--modality"
+        description: |
+          Which modality from the input MuData file to process.
+        type: string
+        default: "rna"
+        required: false
+      - name: "--input_layer"
+        type: string
+        required: false
+        description: "Input layer to use. If None, X is used. This layer must contain raw counts."
+      - name: "--var_gene_name"
+        type: string
+        description: |
+          Column name in the variable features table that contains gene symbols.
+          This is used for reporting gene names in the output.
+        default: "feature_name"
+        required: false
+      - name: "--obs_cell_group"
+        type: string
+        description: |
+          .obs field containing the cell group information, for example per cell type or per cell cluster.
+          If true, performs per-cell-group analysis with cell-group-specific results.
+
+  - name: Arguments
+    arguments:
+      - name: "--design_formula"
+        type: string
+        description: |
+          Design formula for DESeq2 analysis in R-style format.
+          Specifies the statistical model to account for various factors.
+        required: true
+        example: "~ disease + treatment"
+      - name: "--contrast_column"
+        type: string
+        description: |
+          Column in the metadata to use for the contrast.
+          This column should contain the conditions to compare.
+        required: true
+        example: "treatment"
+      - name: "--contrast_values"
+        type: string
+        multiple: true
+        description: |
+          Values to compare in the contrast column.
+          First value is the control group, following values are comparison groups. 
+          Values must be present in the fields specified for the design formula.
+        required: true
+        example: ["ctrl", "stim"]
+      - name: "--filter_genes_min_samples"
+        type: integer
+        min: 1
+        description: |
+          Minimum number of samples a gene must be expressed in to be included in the analysis.
+          If None, no filtering is applied.
+        required: false
+      - name: "--p_adj_threshold"
+        type: double
+        description: |
+          Adjusted p-value threshold for significance.
```

Comment:
Maybe it is relevant to convey how the p-value is corrected for multiple testing. Bonferoni, Benjamini-Hochberg, ...?